### PR TITLE
Added a new field to fill popup content.

### PR DESCRIPTION
**Pull request by Daniel-KM**
I needed to fill the popup content of the markers and polygon, so I added a text area for that.

### CODE REVIEW

**Comment by Daniel-KM** [comment]
I fixed the style.

**Comment by Daniel-KM** [comment]
But not sure if it is a good idea: the textarea can be inserted directly in the popup, so the content can be updated inline and like the visitor will see it.

File: src/javascript/FormElements/PopupContentElement.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    title: 'Description'
+  },
+
+  createContent: function () {
+    let uiElement = this.options.uiElement
+
+    let textArea = L.DomUtil.create('textarea', '', uiElement)
+    this.options.descTextAreaField = textArea // keep the reference
+
+    // L.DomEvent.addListener(textArea, 'blur', function () {
```
Comment (fupduck) [inline src/javascript/FormElements/PopupContentElement.js]: could you remove this line?

File: src/javascript/Form/MarkerForm.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 L.StyleEditor.forms.MarkerForm = L.StyleEditor.forms.Form.extend({
   options: {
     formElements: {
+      'popupContent': L.StyleEditor.formElements.PopupContentElement,
```
Comment (fupduck) [inline src/javascript/Form/MarkerForm.js]: could you move this beneath 'size'?

File: src/javascript/Form/GeometryForm.js [hunk 3]
```diff
@@ -2,6 +2,7 @@
 L.StyleEditor.forms.GeometryForm = L.StyleEditor.forms.Form.extend({
   options: {
     formElements: {
+      'popupContent': L.StyleEditor.formElements.PopupContentElement,
```
Comment (fupduck) [inline src/javascript/Form/GeometryForm.js]: could you move this beneath 'size'?
I presume most users will more often change the icons and not necessarily the popupContents

File: src/javascript/FormElements/PopupContentElement.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+L.StyleEditor.formElements.PopupContentElement = L.StyleEditor.formElements.FormElement.extend({
+
+  options: {
+    title: 'Description'
+  },
+
+  createContent: function () {
+    let uiElement = this.options.uiElement
+
+    let textArea = L.DomUtil.create('textarea', '', uiElement)
```
Comment (fupduck) [inline src/javascript/FormElements/PopupContentElement.js]: could you add the class `leaflet-styleeditor-input` to the textArea?

**Comment by Daniel-KM** [comment]
I followed your remarks and added a style 100% for the textarea. 
I think that it can be improved: when clicked in the field description, the popup may be displayed and the text filled instantly, like other buttons. It would be for a future pull request later, as I don't need it for now.

**Comment by fupduck** [comment]
@Daniel-KM Thanks! I created #62 concerning your last suggestion.

**Pull request merged**